marimbabot_speech/src/speech2text.py

```diff
@@ -46,7 +48,7 @@ def tmp_callback(self, tmp_file_msg):
 		speech_msg.speech = text
 		speech_msg.is_finished = tmp_file_msg.is_finished
 		self.speech_pub.publish(speech_msg)
-		rospy.logdebug(f"speech published.")
+		#rospy.logdebug(f"speech published.")
 
```

Comment:
Please remove the comments or include them as they are just just debug information.

Reply:
why we need to remove the debug term, we can just disable it in future, and it can be useful.  Or we can do this before we release the final version.

Reply:
Considering the commented lines: They are probably commented because they are not needed (except for development) - so we won't need them in main branch as well. Furthermore, there are many reasons like readability, the possibility of comments getting outdated and no one cares about them anymore and so on. :)
Also they are not explanatory and are not TODOs. That is what I would say
@yunlong-wang-cn

Reply:
yes, so currently we are at development version, the final version should be released after integartion test in my oponion.